plugins/inputs/statsd/statsd_test.go

```diff
@@ -1046,7 +1046,7 @@ func TestCachesExpireAfterMaxTTL(t *testing.T) {
 	require.NoError(t, s.parseStatsdLine("valid:45|c"))
 	require.NoError(t, s.Gather(acc))
 
-	// Wait for the metrics to arrive
+	// Wait for the metrics to  arrive
```

Comment:
This seems not relevant for this PR, please revert.

Reply:
> This seems not relevant for this PR, please revert.

Sorry, this is an operation done by packaging, I changed it back